framework/src/multiapps/TransientMultiApp.C

```diff
@@ -364,12 +364,13 @@ TransientMultiApp::solveStep(Real dt, Real target_time, bool auto_advance)
 
               while (!caught_up && catch_up_step < _max_catch_up_steps)
               {
-                Moose::err << "Solving " << name() << "catch up step " << catch_up_step
+                Moose::err << "Solving " << name() << " catch up step " << catch_up_step
```

Comment:
Wait why are we use Moose::err here? This should go to the `_console`. Also, it looks like you sneaked in an extra space.